### ISSUE TITLE
No license?

### ISSUE THREAD

**Issue by jsmaniac** [takikawa/tr-pfds#10]
Hi,

I couldn't find anywhere the license of this package, neither on github nor on the [original PLaneT page](https://planet.racket-lang.org/display.ss?package=pfds.plt&owner=krhari) .

Without any further information, we're in principle not allowed to use the package in any way (although that's most likely not the intent of the original author).

Would it be possible to add a proper LICENSE file, or a line in the docs or in the README? Thanks!

**Comment by takikawa** [comment]
Hey, thanks for bringing that up. I don't know what the original intended license is either, but I'd be happy to add one. @samth did you ever discuss a license with Hari when you two were working on this originally?

**Comment by samth** [comment]
No, we didn't. I'm happy with whatever you choose. @krhari, what do you think?

**Comment by ron-premier** [comment]
It doesn't seem that krhari is active. What is the path forward here?
